dataValidator/helpers/dataLoader/asyncValidateDir.test.js

```diff
@@ -32,7 +32,7 @@ describe('asyncValidateDir helper', () => {
 
   it('should call readScenario for every file in the /data directory', async () => {
     const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
-    await expectMethodToBeCalledTimes(291, readScenarioSpy);
+    await expectMethodToBeCalledTimes(293, readScenarioSpy);
```

Comment:
Tests are currently failing on this value.
```suggestion
    await expectMethodToBeCalledTimes(291, readScenarioSpy);
```